README.md

```diff
@@ -19,3 +19,29 @@ quantitatively reason with them.
 Add your answer to this markdown file. [This
 page](https://docs.github.com/en/get-started/writing-on-github/working-with-advanced-formatting/writing-mathematical-expressions)
 might help with the notation for mathematical expressions.
+
+--------------------------------------------------------------------------------
+
+Using Slide 34 from the sorting powerpoint, we have 3 possible options, the two end quarters of the array being bad, and the middle half being good.  We can check all possible combinations to see what the odds are that we get a bad pivot and see overall what the chances are for getting a good one as a result.
+
+Starting out; lets let Left be the bottom quarter of the array, Right be the top quarter, and Middle be the middle half (abbreviated to L, R, and M respectively as writing out left, right, and middle started getting tedious).  
+
+Now starting with only the left and right, since they have a higher chance of being bad than the middle, we get:  LLL, LLR, LRL, RLL, RRR, RRL, RLR, LRR
+
+Since each L or R have a 1/4 chance of being bad we get a $\frac{1}{4^3} = \frac{1}{64}$ * 8 for each possible combo = $\frac{8}{64}$ $\approx$ = 12.5% of a bad pivot.
+
+Now we have to take combinations of left, right, and middle because even though middle is good almost for sure, the introduction of left and right will lower the chances of a good pivot
+
+So possible combinations are: LLM, LML, MLL, RRM, RMR, MRR
+
+This will have a difference chance however since each of those will be $\frac{1}{4^2}*\frac{1}{2}$ = $\frac{1}{32}$, two bad options and one good option. Next $\frac{1}{32}$ * 6 for six possible combos and we get $\frac{6}{32}$ $\approx$ 18.75% chance for a bad pivot.
+
+Adding these together we get 12.5 + 18.75 = 31.25% for a bad pivot total using this method, which means subracting from 100, we get 100 - 31.25 = 68.75% chance for a good pivot which are pretty good odds all things considered.
+
+
+So far I really only needed the slides for this exercise, it was fairly straight forward probability.
+
+I will say however that I did take a brief gander at a couple other repositories to see if anyone had more streamlined ways to go about it.
```

Comment:
What other repositories did you take a look at, and what did you take from those?

Reply:
I checked Nolans, as well as a guy named Clayton from a different semester but knew from a different class, but I was sitting there just listing out combinations of the left, right, and middle of the graph and thought there had to be a better way, and there wasn't at least from what I saw

Reply:
And what parts of the code did you write yourself and what parts did you get from other repos?

Reply:
I didn't get anything from other people I just felt that what I was doing was clunky and wanted to see if anyone had any better ideas and when what they were doing was already what I was doing I just went about my calculations.  I'll update ny README to better reflect this.